eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/configuration/jobs/CreateSubsnpAccessionsJobConfigurationTest.java

```diff
@@ -66,6 +69,30 @@ public void executeJob() throws Exception {
         assertEquals(EXPECTED_VARIANTS, numVariantsInReport);
     }
 
+    @Test
+    public void executeJobReportOrderCheck() throws Exception {
+        JobExecution jobExecution = jobLauncherTestUtils.launchJob();
+        assertEquals(BatchStatus.COMPLETED, jobExecution.getStatus());
+
+        File output = new File(inputParameters.getOutputVcf());
+        int firstVariantLineNumber = getVariantLineNumberByPosition(output, "20" + "\t" + "60027");
+        int secondVariantLineNumber = getVariantLineNumberByPosition(output, "20" + "\t" + "60028");
+        assertTrue(firstVariantLineNumber < secondVariantLineNumber);
+    }
+
+    private static int getVariantLineNumberByPosition(File output, String position) throws IOException {
+        BufferedReader fileInputStream = new BufferedReader(new InputStreamReader(new FileInputStream(output)));
+        String line;
+        int lineNumber = 0;
+        while ((line = fileInputStream.readLine()) != null) {
+            if (line.startsWith(position)) {
+                return lineNumber;
+            }
+            lineNumber++;
+        }
+        throw new IllegalStateException("VCF didn't have any data lines");
```

Comment:
This message is not accurate. If the VCF has data lines, but the "position" we are looking is not in the VCF, the exception will be thrown as well. Maybe this message is better:
"The VCF does not contain any variant with position " + position